src/Contracts/Models/InformaticsGateway.cs

```diff
@@ -1,16 +1,19 @@
-﻿using Newtonsoft.Json;
+﻿// SPDX-FileCopyrightText: © 2021-2022 MONAI Consortium
+// SPDX-License-Identifier: Apache License 2.0
+
+using Microsoft.Extensions.Configuration;
 
 namespace Monai.Deploy.WorkflowManager.Contracts.Models
 {
     public class InformaticsGateway
     {
-        [JsonProperty(PropertyName = "ae_title")]
+        [ConfigurationKeyName("ae_title")]
```

Comment:
May want to use `JsonProperty` for models instead of `ConfigurationKeyName`

Reply:
changed